addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -70,8 +70,8 @@ export class AnalyticDistribution extends Component {
             fieldString: this.env._t("Analytic Distribution Template"),
         });
         this.allPlans = [];
-        this.lastAccount = this.props.account_field ? this.props.record.data[this.props.account_field] : false;
-        this.lastProduct = this.props.product_field ? this.props.record.data[this.props.product_field] : false;
+        this.lastAccount = this.props.account_field ? this.props.record.data[this.props.account_field] || false : false;
```

Comment:
```suggestion
        this.lastAccount = this.props.account_field && this.props.record.data[this.props.account_field]? this.props.record.data[this.props.account_field] : false;
```
I don't know if we think this is better